src/models/Books.ts

```diff
@@ -0,0 +1,34 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity()
+class Book {
+  @PrimaryGeneratedColumn()
+  uuid!: number
```

Comment:
Usually `uuid` is a string like `9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d`, but its just for you information :)